prow/tide/README.md

```diff
@@ -0,0 +1,131 @@
+# Tide
+
+Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md)
+component for managing a pool of PRs that match a given set of criteria.
+Automatically retesting PRs in the pool and merging them if they pass tests
+against the latest base branch commit.
+
+Configuration of Tide is located under
+[prow/config.yaml](https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml)
+file. It consists of two parts:
+
+  * **Specification of PR pools/criteria**:
+    A set of all PRs Tide keeps track of.
+    The PRs are periodically checked if they are ready to be merged.
+    Configured under `tide` key of the configuration file.
+
+  * **Specification of PR tests**:
+    A set of jobs that are run over each PR before it gets merged.
+    No job is required to be provided in which case it's enough a PR has all
+    relevant labels set. If a set of jobs is specified,
+    all must succeed (unless configured otherwise).
```

Comment:
the "unless configured otherwise" part is still not merged, right? Today, every commit status in a PR needs to be green for tide to merge the PR.

Reply:
What about the `skip_report` field then? Or the `Tide` does not care about the PR status and checks the Prow jobs directly?

Reply:
Yeah, skip_report works fine. Please document it.